generated/nidaqmx/stream_readers.py

```diff
@@ -247,6 +247,44 @@ def read_one_sample(self, timeout=10):
     def read_waveform(
         self, number_of_samples_per_channel=READ_ALL_AVAILABLE, timeout=10
     ) -> AnalogWaveform[numpy.double]:
```

Comment:
Add nitypes to the intersphinx mapping and make sure it hyperlinks `AnalogWaveform` to the nitypes documentation.

https://github.com/ni/nidaqmx-python/blob/master/docs/conf.py#L48

https://nidaqmx-python--798.org.readthedocs.build/en/798/stream_readers.html#nidaqmx.stream_readers.AnalogSingleChannelReader.read_waveform